File: frontend/src/components/models/SignUpModel.js
class SignUpModel {
    constructor(password, password_confirm) {
      this.password = password;
      this.password_confirm = password_confirm;
    }
  
    isPasswordValid() {
      return this.password === this.password_confirm;
    }
  }
  
  export default SignUpModel;
  

File: lib/presentation/sign_up_screen/models/sign_up_model.dart
class SignUpModel {
  String? username;
  String? email;
  String? password;

  SignUpModel({this.username, this.email, this.password});

  SignUpModel.fromJson(Map<String, dynamic> json) {
    username = json['username'];
    email = json['email'];
    password = json['password'];
  }

  Map<String, dynamic> toJson() {
    return {
      'username': username,
      'email': email,
      'password': password,
    };
  }
}
